packages/fxa-settings/public/locales/en-US/settings.ftl

```diff
@@ -11,6 +11,25 @@ nav-connected-services = Connected Services
 nav-paid-subs = Paid Subscriptions
 nav-email-comm = Email Communications
 
+## Password change page
```

Comment:
Add password change localization since I was in that part of the code and needed to test errors localization.

Reply:
This file can also be ignored since its auto generated.